apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-docs.component.ts

```diff
@@ -119,4 +120,8 @@ export class PlatformDynamicPageDocsComponent {
             component: 'PlatformDynamicPageFlexibleColumnExampleComponent'
         }
     ];
+
+    constructor(location: LocationStrategy) {
+        location.onPopState(() => (document.getElementById('page-content').style.overflowY = 'auto'));
```

Comment:
It would be great to have `register` and `unregister` codes in the same place. Can we do that?
Also, this subscription will live forever once we run it.

Reply:
We can create a specific service for that and share it with all `dynamic page` examples.
In order to reset we should just `.overflowY = ''`

Reply:
You're right. I am now listening to `popstate` event and subscribing/unsubscribing to it, and it works the same. Please check.

Reply:
Thanks for adding logic for unsubscribing.

What about the comment to keep `set/reset page overflow` logic in one service (now it leaves in a bunch of example files)?
Also when we do resetting by `style.overflowY = 'auto'` means that we override styles defined in `css`, instead we just need to reset inline-style by `style.overflowY = ''`

Reply:
@dimamarksman do we really need a service for this? It is just something very specific to our documentation and doesn't really add much value to the end-user.  
Sure, i can replace `style.overflowY = 'auto'` with `style.overflowY = ''`,  thanks for explaining it, from the earlier comment I thought setting it to `style.overflowY = ''` was related to the popstate issue. 😅

Reply:
I have made the requested changes, including using a service instead. Please check.